packages/app/src/components/AppNotificationManager/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+import { updateAppActions } from '../../actions';
+import { StyledNotificationContainer } from './styled';
+import AppNotification from './notification';
+import uniqueId from './utils/uniqueId';
+
+const AppNotificationManager = () => {
+    const [notifications, setNotifications] = useState([]);
+
+    const removeNotification = (key) => {
+        setNotifications(notifications.filter((item) => item.key !== key));
+    };
+
+    const addNotification = (notification, timeout = 2000) => {
+        const key = uniqueId('notification');
+        setNotifications([{ key, timeout, ...notification }, ...notifications]);
+        if (timeout && timeout > 0) {
+            setTimeout(() => {
+                removeNotification(key);
+            }, timeout + 600);
```

Comment:
why we need use `timeout + 600`?

Reply:
It is necessary in order to give the transition enough time to finish, this could be done in 2 ways, starting the transition on timeout and wait 600ms (duration of the transition) to remove the component from the tree, or start the transition 600ms before the timeout and remove the component from the tree on timeout, I went with the first option, but this can be changed easily